http/src/main/resources/leo.conf

```diff
@@ -6,10 +6,6 @@ application {
   leoUrlBase = ${?LEO_URL_BASE}
 }
 
-app-service {
```

Comment:
hmmm looks like a merge fail possibly, will investigate (i.e. I did not remove this)